test/e2e/tree-shaking-flow/fixture/node_modules/fixture-pkg/dist/index.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
```

Comment:
We couldn't have `export type` declarations in the dependency's entry point, so maybe this test isn't the best...

Currently using the transpiled `index.js` here and added an `index.js.flow` file for Flow/IDE integration.